lib/archethic_web/templates/dir_listing/index.html.heex

```diff
@@ -116,6 +116,7 @@
         <thead>
             <tr>
                 <th>Name</th>
+                <th class="size">Size(Bytes)</th>
```

Comment:
you need to define the css class called size.

Reply:
I will update that. No problem with it. Please tell me how to view the below screen in localhost.

Reply:
hello, you can follow this doc https://github.com/archethic-foundation/archethic-node/issues/800#issuecomment-1371250668 to view the screen